addons/web_tour/static/src/tour_service/tour_automatic.js

```diff
@@ -70,6 +70,11 @@ export class TourAutomatic {
                             );
                         },
                         action: async () => {
+                            try {
```

Comment:
I'd wrap that in a in if to avoid the extra microtick in the usual case

Reply:
The if is in the method